strapi/strapi
# feat(review-workflows): add stage attribute in entities

### What does it do?

In this PR, there are multiple additions:
- Possibility to include an `reviewWorkflows` options in any content-type
- A new stage attribute in the entities where reviewWorkflows is activated
- A migration of entities when reviewWorkflows is being activated

### How to test it?

You can start your Strapi project with the admin panel. Go into the content type builder, activate review workflow on any content-type.
Then it should migrate all entities inside this content-type and add a relation to the first stage of the default workflow.
You can retrieve this stage on any entity in the content-type by populating `strapi_reviewWorkflows_stage`.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
